frontends/ol-components/src/components/TiptapEditor/extensions/node/Image/ImageWithCaption.tsx

```diff
@@ -196,7 +246,13 @@ export function ImageWithCaption({
           </div>
         )}
 
-        <Image src={src} alt={alt || caption} layout={layout} />
+        <Image
+          src={src}
+          alt={alt || caption}
```

Comment:
A screenreader will read the alt text then the caption, so using the caption as a fallback for the alt text doesn't seem beneficial to me. It would just result in the caption being read twice.

I'd leave this as `alt={alt}`; if there's no alt text...well, at least it will read the caption.

(That said, at some point, we should figure out how to make alt text required.)